src/BSML/Components/CustomListTableData.cpp

```diff
@@ -98,7 +98,7 @@ namespace BSML {
         auto tableCell = reinterpret_cast<BSML::BoxTableCell*>(tableView->DequeueReusableCellForIdentifier(reuseIdentifier));
 
         if (!tableCell) {
-            if (!levelPackCell || !Object::IsNativeObjectAlive(levelPackCell)) {
+            if (!levelPackCell || !levelPackCell->m_CachedPtr.m_value) {
```

Comment:
Shouldn't this be SafePtrUnity aswell?

Reply:
could be but they are already C# fields so I don't see the need?

Reply:
Yes it's fine I didn't see it's a custom type